fbfmaproom/fbfmaproom.py

```diff
@@ -184,6 +202,39 @@ def format_enso(x):
     assert False, f"Unknown enso state {x}"
 
 
+format_funcs = {
+    'number0': number_formatter(0),
+    'number1': number_formatter(1),
+    'number2': number_formatter(2),
+    'number3': number_formatter(3),
+    'number4': number_formatter(4),
+    'timedelta_days': format_timedelta_days,
+    'bad': format_bad,
+    'enso': format_enso,
+}
```

Comment:
I removed the `year` formatter from this dictionary. It was a closure over a local variable of `table_columns`, which made reorganizing things complicated. I eventually realized that the `year` formatter is only used once, for a column that's already a special case anyway, so there was no benefit to keeping it in the dictionary. Now `format_funcs` and `make_column` can be defined at the top level instead of inside `table_columns()`.